### ISSUE TITLE
main branch has not been updated

### ISSUE THREAD

**Issue by jpmckinney** [OP-TED/eForms-SDK#361]
I would have expected it to match the recently released 1.6.0.

**Comment by bertrand-lorentz** [comment]
Due technical issues on Maven Central, we haven't been able to publish version 1.6.0 there.
Once this is done, the branch will be merged into main.

**Comment by bertrand-lorentz** [comment]
Version 1.6.0 is now available on Maven Central, so the main branch has been updated.
